clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml

```diff
@@ -0,0 +1,62 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: node-observability-operator
```

Comment:
registry.ci.openshift.org/ocp/4.11:node-observability-agent

If the image is promoted to ocp, why do we need the namespace?

Reply:
Created a new organization in quay.io for Node Observability Operator. Changed the promotion of the agent image from OCP to the new organization.